Sources/ACKategories/SwiftUIExtensions/SwiftUILayoutGuides.swift

```diff
@@ -0,0 +1,403 @@
+import SwiftUI
+
+// Source: https://github.com/tgrapperon/swiftui-layout-guides
+/// This view populates its content's ``layoutMarginsInsets`` and ``readableContentInsets``.
+public struct WithLayoutMargins<Content>: View where Content: View {
+    let content: (EdgeInsets) -> Content
+
+    /// Initialize a ``WithLayoutMargins`` view, populating its content's ``layoutMarginsInsets``
+    ///  and ``readableContentInsets``.
+    ///
+    /// - Parameter content: A closure that builds a `Content` view from the layout
+    /// margins provided in the form of an `EdgeInsets` argument.
+    public init(@ViewBuilder content: @escaping (EdgeInsets) -> Content) {
+        self.content = content
+    }
+
+    /// Initialize a ``WithLayoutMargins`` view, populating  its content's ``layoutMarginsInsets``
+    /// and ``readableContentInsets``.
+    ///
+    /// - Parameter content: A closure that builds a `Content` view.
+    public init(@ViewBuilder content: @escaping () -> Content) {
+        self.content = { _ in content() }
+    }
+
+    public var body: some View {
+        InsetContent(content: content)
+            .measureLayoutMargins()
+    }
+
+    private struct InsetContent: View {
+        let content: (EdgeInsets) -> Content
+        @Environment(\.layoutMarginsInsets) var layoutMarginsInsets
+        var body: some View {
+            content(layoutMarginsInsets)
+        }
+    }
+}
+
+/// This view makes its content `View` fit the readable content width.
+///
+/// - Note: This modifier is equivalent to calling ``.fitToReadableContentWidth()`` on
+/// the content view.
+@available(
+  iOS, deprecated: 9999.0, message: "Use the `.fitToReadableContentWidth` modifier instead."
+)
+@available(
+  macOS, deprecated: 9999.0, message: "Use the `.fitToReadableContentWidth` modifier instead."
+)
+@available(
+  tvOS, deprecated: 9999.0, message: "Use the `.fitToReadableContentWidth` modifier instead."
+)
+@available(
+  watchOS, deprecated: 9999.0, message: "Use the `.fitToReadableContentWidth` modifier instead."
+)
+public struct FitReadableContentWidth<Content>: View where Content: View {
```

Comment:
I would rather not add a code that is already marked as deprecated, can we remove it?